diff --git a/lib/core/migrator.js b/lib/core/migrator.js
index 0fcc51a157e10891e55b814b27ad5f00a4ea315d..9dd2e2ddb8efcbd76572bd5cd2b1a874914f6057 100644
--- a/lib/core/migrator.js
+++ b/lib/core/migrator.js
@@ -49,7 +49,7 @@ async function getMigrator(type, args) {
     migrations: {
       params: [sequelize.getQueryInterface(), Sequelize],
       path: _index.default.path.getPath(type),
-      pattern: /^(?!.*\.d\.ts$).*\.(cjs|js|cts|ts)$/
+      pattern: /^(?!.*\.d\.ts$).*\.(cjs|mjs|js|cts|ts)$/
     }
   });
   return sequelize.authenticate().then(() => {
